Assets/Scripts/Interactable.cs

```diff
@@ -15,7 +16,12 @@ public struct IrritationReaction
   public List<IrritationReaction> irritationReactionList = new List<IrritationReaction>();
   NonPlayerController reactingNPC;
 
-  public void HandleInteraction()
+  void Start()
+  {
+    interactableUI = GetComponent<InteractableUI>();
```

Comment:
Thinking we should only ever access the `InteractableUI` through the `Interactable` class